test/recipes/95-test_external_rpki-client-portable.t

```diff
@@ -0,0 +1,29 @@
+#! /usr/bin/env perl
+# Copyright 2024 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the Apache License 2.0 (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+
+use OpenSSL::Test;
+use OpenSSL::Test::Utils;
+use OpenSSL::Test qw/:DEFAULT data_file srctop_file bldtop_dir/;
+use Cwd qw(abs_path);
+
+setup("test_external_rpki-client-portable");
+
+plan skip_all => "No external tests in this configuration"
+    if disabled("external-tests");
+plan skip_all => "rpki-client-portable not available"
+    if ! -f srctop_file("rpki-client-portable", "configure.ac");
```

Comment:
This check makes sense, but I don't see that rpki-client-portable has ever been added as  part of our submodules, so i'm not sure in what conditions this will ever get run in CI without manual intervention.  Is there another PR that adds this submodule somewhere?

Reply:
I've just added the test and gave it a try to run it manually. The idea is that we should run it as a part of pre-release test.

I will take a look at other external tests we have and add the missing pieces. Thank you for pointing it out.